modules/apps/frontend-icons/frontend-icons-clay/build.gradle

```diff
@@ -0,0 +1,23 @@
+import com.liferay.gradle.util.copy.StripPathSegmentsAction
+
+task buildClayIcons(type: Copy)
+
+buildClayIcons {
```

Comment:
:thinking: I wonder if we would be able to do this with the bundler instead. Instead of packaging the icons with gradle, let the bundler run and, at the same time, provide a JS AMD module to retrieve the icons (as we do with CSS files). That would be a way to let them be used from JS without any need to provide a new API.

However, I'm just proposing something to explore/investigate, not saying that I like it better than referring to the icons by their URL in the server. I don't know what would be the pros and cons of such approach...